File: Latihan B/php/Disk.php
<?php
// deklarasi kelas
class Disk{
	//variabel lokal
	private $capacity;
	private $type;
    private $price;
	//konstruktor
	public function __construct($capacity = "", $type = "", $price = 0){
		$this->capacity = $capacity;
        $this->type = $type;
        $this->price = $price;
	}
	//get dan set setiap variabel
	public function setprice($price){
		$this->price = $price;
	}
	public function getprice(){
		return $this->price;
	}
	public function setcapacity($capacity){
		$this->capacity = $capacity;
	}
	public function getcapacity(){
		return $this->capacity;
	}
    public function settype($type){
		$this->type = $type;
	}
	public function gettype(){
		return $this->type;
	}
	// destruktor
	public function __destruct(){
	}
}
?>

File: Latihan A/cpp/Disk.cpp
// Library
#include <iostream>
#include <string>
using namespace std;
// deklarasi kelas
class Disk{
    private:
    // variabel private
        string type;
        string capacity;
        int price;
    // variabel dan method publik
    public:
        // konstruktor
        Disk(){

        }
        Disk(string type, string capacity, int price){
            this->type = type;
            this->capacity = capacity;
            this->price = price;
        }
        // get dan set
        void settype(string type){
            this->type = type;
        }
        string gettype(){
            return this->type;
        }
        void setcapacity(string capacity){
            this->capacity = capacity;
        }
        string getcapacity(){
            return this->capacity;
        }
        void setprice(int price){
            this->price = price;
        }
        int getprice(){
            return this->price;
        }
        // destruktor
        ~Disk(){}
};
